### ISSUE TITLE
remove subscriber lock for loading card images in deck editor

### ISSUE THREAD

**Issue by brine** [octgn/OCTGN#1682]
this has been discussed as a possibility.  Some games don't have reliable image packs (for many possible reasons) and restricting this feature for users who may need to get the images on their own isn't beneficial.

**Comment by darkhax** [comment]
This has been a very big issue for Magic: The gathering. I have been unable to download any image packs since the 23rd.

**Comment by darkhax** [comment]
Have there been any updates for this issue? The plugin is still not working. 

I have found that users can still manually import images. Here are the steps for that. 
1. Download the .o8c image pack files online. These can usually be found pretty easily.
2. Open the .o8c file using 7Zip, WinRar or a similar archive manager. If you don't have an archive manager you can rename the file from .o8c to .zip and then open it with your basic file explorer. 
3. Find the Octgn image database folder. This is probably in `C:\Users\YourName\Documents\Octgn\ImageDatabase`
4. Copy the folder from the .o8c file to the ImageDatabase folder
5. If a message pops up saying you have existing files with the same name, you probably shouldn't override them.

**Comment by brine** [comment]
Why are you opening the o8c files?  OCTGN has an install images button in the games manager

**Comment by darkhax** [comment]
Interesting, everyone I have been playing with overlooked that button. We tried to import images from the deck editor, which gives the subscriber only message.

**Comment by brine** [comment]
Yeah its been the default way to install sets for many years now, I’ve only recently started developing image downloader plugins for certain games
